features.ts

```diff
@@ -12,14 +12,6 @@ export const getFeatureFlags = ({
     'feature-flags-implementation-proof': {
       isActive: environmentName === 'development',
     },
-    // FEATURE-FLAG-EXPIRES [2021-08-31]: person-timeline
-    'person-timeline': {
-      isActive: environmentName === 'development',
```

Comment:
Typically to 'release' a feature flag you keep the flag in place, and just change the `isActive` logic to `true`. That is the least destructive, lowest code change, and lowest risk way to launch a new feature. We then later come back and tidy up the flags etc.

In this specific case it's small enough not to really matter, but probably worth thinking about for the future.

Reply:
Right, that makes sense. Will keep in mind for next release.